src/indra_cogex/apps/queries_web/__init__.py

```diff
@@ -106,14 +279,39 @@ def get_example_data():
     # Example for /gene/discrete
     "gene_list": fields.List(fields.String, example=EXAMPLE_GENE_IDS),
     # Examples for positive_genes and negative_genes for /gene/signed
-    "positive_genes": fields.List(fields.String,example=EXAMPLE_POSITIVE_HGNC_IDS),
-    "negative_genes": fields.List(fields.String,example=EXAMPLE_NEGATIVE_HGNC_IDS),
+    "positive_genes": fields.List(fields.String, example=EXAMPLE_POSITIVE_HGNC_IDS),
+    "negative_genes": fields.List(fields.String, example=EXAMPLE_NEGATIVE_HGNC_IDS),
     "gene_names": fields.List(fields.String, example=continuous_analysis_example_names),
     "log_fold_change": fields.List(fields.Float, example=continuous_analysis_example_data),
     "species": fields.String(example="human"),
     "permutations": fields.Integer(example=100),
     "source": fields.String(example="go"),
     "indra_path_analysis": fields.Boolean(example=False),
+    # Cell marker
+    "cell_type": fields.List(fields.String, example=["cl", "0000020"]),
+    "marker": fields.List(fields.String, example=["hgnc", "11337"]),
+    # Pubmed
+    "publication": fields.List(fields.String, example=["pubmed", "14334679"]),
+    "journal": fields.List(fields.String, example=["nlm", "0000201"]),
```

Comment:
Try to find an example that returns a value, this example returns an empty result for me.

Reply:
I think these examples do return values.  I ran a test for these examples. 
marker : ["hgnc", "11337"] returns Cell Type: ["cl", "0000020"] 

publication : ["pubmed", "14334679"] returns  Journal result: [(:Journal { id:'NLM:0000201', issn_l:'0567-8064', name:'Acta radiologica: therapy, physics, biology', abbr_title:'Acta Radiol Ther Phys Biol' })]. 

journal : ["nlm", "0000201"] returns several publications from journal articles.

Reply:
I meant to only comment on the `nlm` example. I think the minimum number of lines shown is 4, so I selected line 295 for the nlm id, but line 292-294 are also shown.

I checked again, and this example does indeed return a result for `/api/get_publications_for_journal`, but it does not for `/api/get_publisher_for_journal`, if you can find an example NLM ID that works for both these endpoints, that would be great, but don't spend too much time on it.